### PR TITLE
Enable concatination of any Filetype. 

### PR DESCRIPTION
**Pull request by alwin-rewedigital**
Clean Version of #64
This will fix: #40

Now it should be the Way you prefered. Please take a closer look to the Readme, it thats fine for you.

Greets

### CODE REVIEW

File: lib/configTools.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+/**
+ * Returns all dest, cssDest, jsDest, scssDest ... Configs
+ *
+ * @param an Array of possible configs wich should be filtered and extracted
+ *
+ * @return the Config prefix (or FileExtension). For example css for cssDest, or js for dest.
+ */
+function extractDestData(data) {
+
+	if (destinationConfigExists) {
```
Comment (sapegin) [inline lib/configTools.js]: Looks like destinationConfigExists is a function. Is this correct?

File: Readme.md [hunk 2]
```diff
@@ -38,18 +38,25 @@ Concatenated files will be joined on this string. If you're post-processing conc
 
 ### Parameters
 
-
 #### dest
 
-Type: `String`, optional (at least one of [`dest`, `cssDest`] must exist).
+Type: `Object`, required. Defines the Filetypes to be concatenated into a destination File.
+Where the key is the Fileextension (without the dot) and the value is the destination File.
+E.g:
```
Comment (sapegin) [inline Readme.md]: Are you German? Why do you start nouns from capital letters? ;-)

**Comment by sapegin** [comment]
Otherwise looks good enough. (There are code style issues also but I’m too lazy to describe all of them.)

File: Readme.md (hunk 2 above)
Comment (alwin-rewedigital) [inline Readme.md]: Yes. I'm not good at this case topic. Camel case, german case, english case....

Comment (sapegin) [inline Readme.md]: At least you don’t need to learn German ;-)

**Pull request merged**